packages/seda-chain-contracts/src/data_request.rs

```diff
@@ -79,12 +102,12 @@ pub mod data_requests {
 
         Ok(Response::new()
             .add_attribute("action", "post_data_request")
-            .add_attribute("dr_id", args.dr_id))
+            .add_attribute("dr_id", posted_dr.dr_id))
     }
 
     /// Returns a data request from the pool with the given id, if it exists.
     pub fn get_data_request(deps: Deps, dr_id: Hash) -> StdResult<GetDataRequestResponse> {
```

Comment:
Same thing here. I think the `may_load` function could take a reference to `dr_id`, which means we could pass it by reference. But not 100% sure.

Reply:
Same here, unfortunately, it doesn't accept a reference.